### PR TITLE
duck 18004

### PR DESCRIPTION
**Pull request by dkocher**
We have updated our build to use the SHA-256 checksum in future versions.

### CODE REVIEW

File: Library/Formula/duck.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Duck < Formula
   desc "Command-line interface for Cyberduck (a multi-protocol file transfer tool)"
   homepage "https://duck.sh/"
-  url "https://dist.duck.sh/duck-src-4.7.1.17911.tar.gz"
-  sha256 "51cf135a12743c0a524c23adeab13f01a635999f8b03122eaf06056e320d365d"
+  url "https://dist.duck.sh/duck-src-4.7.2.18004.tar.gz"
+  sha1 "870903f2988bdde6195a4e6cf754f488b724a49f"
```
Comment (bfontaine) [inline Library/Formula/duck.rb]: Please use a `sha256` here.

Comment (dkocher) [inline Library/Formula/duck.rb]: This is now fixed with 9d9b5b47bd2e92f1d7e98b017c7888c19c7931c1.

**Comment by dunn** [comment]
Looks like this needs rebasing.

**Comment by dkocher** [comment]
I can't seem to resolve this build failure. My branch is up to date with the base branch.

**Comment by dunn** [comment]
From your `cyberduck` branch, did you run `git rebase origin/master`?

**Comment by dkocher** [comment]
Yes, that is what I did with the last push.

**Comment by mikemcquaid** [comment]
@dkocher Squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!

**Pull request closed without merging**